tests/pipelines/animatediff/test_animatediff.py

```diff
@@ -126,6 +132,36 @@ def get_dummy_inputs(self, device, seed=0):
         }
         return inputs
 
+    def test_from_pipe_consistent_config(self):
+        assert self.original_pipeline_class == StableDiffusionPipeline
+        original_repo = "a-r-r-o-w/tinier-stable-diffusion-pipe"
```

Comment:
I have invited you to https://huggingface.co/hf-internal-testing. So you should be able to push it directly there.